core/src/stores/migrations/20240809_backfill_unconnected_tables.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE "tables"
```

Comment:
Assuming we can do this since the migration is fully done, right? So the 60+ tables from GDrive without any parents will also be updated, you confirm?

Reply:
yes, they will be updated too . the migration is over and I just have these 60 tables remaining. Rerunning the migration on them does change anything, as we don't have the parents information in connectors.